agent/VERSION

```diff
@@ -1 +1 @@
-0.72.0
```

Comment:
Since this is on `main`, maybe we should consider just bumping to `0.73`. We need to add the `--relay` soon to support a Server 1.0 release and I'm not sure a 0.72.1 before that makes sense. ??

Reply:
We are at 0.72.1 now. We generally update the version just before a release. In general, we have tried (not always successfully) to make the release tag point to a commit that just bumps the version.

Reply:
I guess I never really paid that much attention before; but on the server side we have 1.0 because that's what main is working *towards*, rather than duplicating the value on the last release branch. It seems odd that Agent would follow a different convention. That is, we've built and released 0.72.1, and the code on `main` is not the same; so wouldn't it be better to build it as something newer?

Reply:
I agree that the `main` branch version should be greater than any shipping version.  Given that we've shipped v0.72.1, the next version _on that branch_ will (presumably) be v0.72.2...so what's on `main` shouldn't be anything less than 0.73, so that if we get builds from each of them there cannot be any confusion.  (If we decide to release something new, it'll either be backported from `main` and released from the `b0.72` branch (in which case it will be labeled v0.72.x) or we'll cut a new branch from `main` (which could be v0.72.x but would more likely be v.73.0, in which case we would/should bump the `main` branch to v0.74 at that time).)

So, yeah, I vote for `main` carrying a version of at least 0.73, but I wouldn't be opposed to it having version 1.0 (which we would set back to 0.73 for the next Agent release, unless it really ends up being 1.0...).